shell/common/rasterizer.h

```diff
@@ -101,6 +107,7 @@ class Rasterizer final : public SnapshotDelegate {
   std::unique_ptr<flutter::CompositorContext> compositor_context_;
   std::unique_ptr<flutter::LayerTree> last_layer_tree_;
   fml::closure next_frame_callback_;
+  bool user_override_resource_cache_bytes_;
```

Comment:
It may just be more future-proof to make this `std::atomic<bool>` in case that someone mistakenly modified it on other threads (than the GPU thread) in a future PR. Considering the low frequency of reading/writing this bit, the overhead of that atomic bool should be negligible.

Reply:
The rasterizer is only accessed on the GPU thread.  I'd prefer not to imply that any part of the rasterizer is safe to access from other threads.

Reply:
Oh, this is in `Rasterizer`, not `Shell`. Ignore more earlier comment :)

Reply:
How about if I check that the method using it runs on the GPU thread?

Reply:
Don't worry about it @dnfield . All fields of `Rasterizer` are only safe inside the GPU thread. So there's little reason to just guard this single field by making your change more complex.

Reply:
Nevermind, I see what you're saying - just switched it to std::atomic<bool>

Reply:
Haha - Github did not refresh on me.  Ok, reverted back.